<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>친구 이름 입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Friend_Name_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Friend_Name_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Friend_Accept_Button">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>129</y>
     <width>137</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>친구 수락</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Friend_Search_Button">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>129</y>
     <width>137</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>친구 검색</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Friend_List">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>271</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
